openmetadata-docs/content/menu.md

```diff
@@ -169,341 +489,26 @@ site_menu:
-  - category: OpenMetadata / Connectors / Pipeline / Fivetran / CLI
-    url: /openmetadata/connectors/pipeline/fivetran/cli
-  - category: OpenMetadata / Connectors / Pipeline / Dagster
-    url: /openmetadata/connectors/pipeline/dagster
-  - category: OpenMetadata / Connectors / Pipeline / Dagster / Airflow
-    url: /openmetadata/connectors/pipeline/dagster/airflow
-  - category: OpenMetadata / Connectors / Pipeline / Dagster / CLI
-    url: /openmetadata/connectors/pipeline/dagster/cli
-
-  - category: OpenMetadata / Connectors / ML Model
-    url: /openmetadata/connectors/ml-model
-  - category: OpenMetadata / Connectors / ML Model / ML Flow
-    url: /openmetadata/connectors/ml-model/mlflow
-  - category: OpenMetadata / Connectors / ML Model / ML Flow / Airflow
-    url: /openmetadata/connectors/ml-model/mlflow/airflow
-  - category: OpenMetadata / Connectors / ML Model / ML Flow / CLI
-    url: /openmetadata/connectors/ml-model/mlflow/cli
-
-  - category: OpenMetadata / Connectors / Metadata
-    url: /openmetadata/connectors/metadata
-  - category: OpenMetadata / Connectors / Metadata / Amundsen
-    url: /openmetadata/connectors/metadata/amundsen
-  - category: OpenMetadata / Connectors / Metadata / Atlas
-    url: /openmetadata/connectors/metadata/atlas
-
-  - category: OpenMetadata / Connectors / Managing Credentials
-    url: /openmetadata/connectors/credentials
-
-  - category: OpenMetadata / Ingestion
-    url: /openmetadata/ingestion
-  - category: OpenMetadata / Ingestion / Workflows
-    url: /openmetadata/ingestion/workflows
-  - category: OpenMetadata / Ingestion / Workflows / Metadata
-    url: /openmetadata/ingestion/workflows/metadata
-
-  - category: OpenMetadata / Ingestion / Workflows/ Metadata / DBT
-    url: /openmetadata/ingestion/workflows/metadata/dbt
-  - category: OpenMetadata / Ingestion / Workflows/ Metadata / DBT / Ingest DBT UI
-    url: /openmetadata/ingestion/workflows/metadata/dbt/ingest-dbt-ui
-  - category: OpenMetadata / Ingestion / Workflows/ Metadata / DBT / Ingest DBT from Workflow Config
-    url: /openmetadata/ingestion/workflows/metadata/dbt/ingest-dbt-workflow-config
-  - category: OpenMetadata / Ingestion / Workflows/ Metadata / DBT / Ingest Owner from DBT
-    url: /openmetadata/ingestion/workflows/metadata/dbt/ingest-dbt-owner
-
-  - category: OpenMetadata / Ingestion / Workflows / Usage
-    url: /openmetadata/ingestion/workflows/usage
-  - category: OpenMetadata / Ingestion / Workflows / Usage / Usage Workflow Through Query Logs
-    url: /openmetadata/ingestion/workflows/usage/usage-workflow-query-logs
-  - category: OpenMetadata / Ingestion / Workflows / Lineage
-    url: /openmetadata/ingestion/workflows/lineage
-  - category: OpenMetadata / Ingestion / Workflows / Profiler
-    url: /openmetadata/ingestion/workflows/profiler
-  - category: OpenMetadata / Ingestion / Workflows / Profiler / Metrics
-    url: /openmetadata/ingestion/workflows/profiler/metrics
```

Comment:
Hi @harshach is deleting this chunk intended?

CI failing with `This slug: openmetadata/ingestion/workflows/profiler/metrics doesn't have a corresponding entry in menu.md.`

Reply:
@pmbrull didn't deleted I just replaced the menu structure.  You'll see at the top
  - category: Connectors / Ingestion / Workflows / Profiler
    url: /connectors/ingestion/workflows/profiler
  - category: Connectors / Ingestion / Workflows / Profiler / Metrics
    url: /connectors/ingestion/workflows/profiler/metrics